docs/guide/advocacy.md

```diff
@@ -0,0 +1,89 @@
+
+* Many changes might change the devDependency tree size, but not impact the bundle
+* Swapping out direct usage of a dependency that is also deeply required might result in a net bigger bundle
+
+In the latter case, you might consider prioritizing working on the upstream consumers of that dependency first.
+
+### Dependency tree
+
+If a dependency cleanup is meant to reduce the size of a project's dependency tree, show the actual change to the tree.
+
+An isolated dependency graph (e.g. via `npmgraph.js.org`) can be useful to give a quick overview of the difference between two dependencies.
+However, sometimes it is not representative of how the overall project's dependency tree will be affected.
+This can be for various reasons, such as the project pulling the same dependencies in elsewhere.
+
+Showing a project dependency graph before and after can be more beneficial for this reason, and may help you find occurrences of target packages you weren't aware of.
+See also [Bundle Size](#bundle-size).
+
+### Speed
+
+If a cleanup is meant to improve the performance of a project, showing actual performance measurements for the project would be the best option followed by sharing a more general benchmark together with an explanation of why it is relevant such as the benchmarked method being called in a hot code path.
+Performance-oriented developers generally don't believe any comparisons not backed up with a reproducible experiment.
+
+Isolated benchmarks of a new dependency can be useful when the comparison is large (i.e. you're comparing against a _much_ slower dependency).
+However, in cases where the gap is small, it is often better to benchmark the project before and after instead of the dependency itself in isolation.
+
+Many maintainers just prefer a project benchmark too, so always refer to the contribution guide or ask the maintainers what their preference is.
+Downstream users often have very different usage scenarios and/or don't use some dependencies in hot paths.
+
+## What Isn't Convincing
+
+Most open source maintainers regularly deal with drive-by contributions that ignore their workflows and do a poor job of explaining themselves.
+Be careful not to make the common mistakes of those contributions, as that weakens the case for your change and for e18e as a whole.
+
+### Abstract Reports
+
+Don't be vague.
+Indicating that a cleanup could offer bug fixes, improved developer experience, or other features in the abstract is not convincing without any evidence those impact the project.
+
+* For bug fixes, be sure to have a reproduction -- much like any other kind of bug report
+* For developer experience and other features, you'll similarly want to be able to demonstrate why the new way is tangibly better
+
+### Downloads and Stars
+
+Non-user-facing metrics such as GitHub Stars and npm download statistics are not generally convincing.
+Both can easily be gamed and thus are not generally seen as reliable metrics.
```

Comment:
"no generally seen as" seems slightly strong. I think they're generally trustworthy and accepted - though I do clearly agree they can be gamed and sometimes are
```suggestion
Both can easily be gamed and thus may not be seen as reliable metrics.
```